include/vcpkg/commands.install.h

```diff
@@ -60,12 +60,6 @@ namespace vcpkg
         const Path& listfile() const;
     };
 
-    ExtendedBuildResult perform_install_plan_action(const VcpkgCmdArguments& args,
```

Comment:
Note that this function is neither called nor implemented. (A function of the same name in `commands.install.cpp` is static and has a different number of arguments)